package.json

```diff
@@ -114,13 +114,15 @@
     "html-loader": "0.5.5",
     "html-to-react": "1.3.3",
     "html-webpack-plugin": "3.2.0",
-    "i18n-calypso": "2.0.1",
+    "i18n-calypso": "file:packages/i18n",
     "ignore-loader": "0.1.2",
     "immutability-helper": "2.8.1",
     "immutable": "3.7.6",
     "imports-loader": "0.8.0",
     "inherits": "2.0.3",
+    "interpolate-components": "1.1.1",
     "is-my-json-valid": "2.19.0",
+    "jed": "1.0.2",
```

Comment:
Shouldn't `interpolate-components` and `jed` continue to be dependencies of the i18n package and not calypso proper? They are listed in the i18n package.json

Reply:
Yeah, but that might need lerna in place. I didn't want to rebase on the lerna PR (yet).